.eslintrc

```diff
@@ -11,6 +11,9 @@
     "extends": [
         "eslint:recommended"
     ],
+    "ignorePatterns": [
+        "node_modules/"
+    ],
```

Comment:
I would change the `argsIgnorePattern` rule to have the following RegEx:

    ^(_|((request|response|next)$))